README.md

```diff
@@ -1,59 +1,72 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
+# Concert Page - Module 1 Capstone Project
 
-> One paragraph statement about the project.
+This project consists in building an event page based on an [online website for a conference](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015). I chose to build it about a fictional Jazz Festival: Jazzaholic, which happens every year in Bogotá, Colombia. The pictures are taken from the internet and the text is made up, so they don't represent a real event in any way.
+
+## Mobile
 
 ![screenshot](./app_screenshot.png)
 
-Additional description about the project and its features.
+## Desktop
+
+![screenshot](./app_screenshot_desktop.png)
 
 ## Built With
 
-- Major languages
-- Frameworks
-- Technologies used
+- HTML5
+- CSS3
+- Bootstrap
+- Visual Studio Code
+- Git & Github
 
 ## Live Demo
 
-[Live Demo Link](https://livedemo.com)
-
```

Comment:
- [ ] You are supposed to use Github pages to deploy your project. Please kindly fix that.

Reply:
Hello again! It was fixed: If you look to the right of the screen, you can see that I had changed it.

![image](https://user-images.githubusercontent.com/70329406/132731918-e353bb37-881d-4141-957c-4c4082e67184.png)

Reply:
This issue is well resolved! ✔️